units/spawnhost_expiration_warning.go

```diff
@@ -0,0 +1,91 @@
+package units
```

Comment:
Please test this code.

Reply:
Almost everything called is unit tested already, but I added a couple of tests